docs/guardrails/scp-guardrails.md

```diff
@@ -2,66 +2,64 @@
 layout: default
 ---
 
-
 # Table of Contents
-* [Prevent account region enable and disable actions](#scp-account-1)
-* [Prevent billing modification actions](#scp-billing-1)
-* [Prevent modifications to specific CloudFormation resources](#scp-cloudformation-1)
-* [Prevent modifications to specific CloudTrails](#scp-cloudtrail-1)
-* [Prevent deleting specific CloudWatch Log groups and streams](#scp-cloudwatch-1)
-* [Prevent enabling and disabling AWS Config](#scp-config-1)
-* [Prevent modifications to tagged AWS Config rules](#scp-config-2)
-* [Prevent disabling default EBS encryption](#scp-ec2-1)
-* [Prevent Creating Default VPC and Subnet](#scp-ec2-2)
-* [Prevent Glacier Deletion](#scp-glacier-1)
-* [Prevent disabling and modifying GuardDuty](#scp-guardduty-1)
-* [Prevent the root user from performing any actions.](#scp-iam-1)
-* [Prevent creating access keys for the root user.](#scp-iam-2)
-* [Prevent modifications to specific IAM roles.](#scp-iam-3)
-* [Prevent iam:UpdateAssumeRolePolicy on specific IAM roles.](#scp-iam-4)
-* [Prevent specific IAM actions](#scp-iam-5)
-* [Prevent KMS Key Deletion](#scp-kms-1)
-* [Prevent Modifications to Specific Lambda Functions](#scp-lambda-1)
-* [Prevent organization leave, delete, or remove actions](#scp-organizations-1)
-* [Prevent sharing resources to accounts outside your organization](#scp-ram-1)
-* [Prevent disabling S3 account public access block](#scp-s3-1)
-* [Prevent S3 unencrypted object uploads](#scp-s3-2)
-* [Prevent S3 public object access](#scp-s3-3)
-* [Prevent Specific S3 Buckets from Deletion](#scp-s3-4)
-* [Prevent Access to Specific S3 Buckets](#scp-s3-5)
-* [Prevent Modifications to Specific SNS Topics](#scp-sns-1)
-
```

Comment:
Need to add row to ToC - * ["Prevent Modifications to EMR Block Public Access] (#scp-emr-1)